pulseguardian/web.py

```diff
@@ -350,38 +350,78 @@ def update_info():
     pulse_username = request.form['pulse-user']
     new_password = request.form['new-password']
     password_verification = request.form['new-password-verification']
+    new_owner_list = _clean_owners_str(request.form['owners-list'])
```

Comment:
True true...  How about ``new_owners`` and ``old_owners`` and then change what used to be ``new_owners`` to be ``new_owner_users`` since it's a list of ``User`` objects?